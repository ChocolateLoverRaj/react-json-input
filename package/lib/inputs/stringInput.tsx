import { Input, InputComponent } from '../props'

const StringInputComponent: InputComponent<string> = props => {
  return null
}

const stringInput: Input = {
  name: 'string',
  Component: StringInputComponent,
  isValid: schema => schema.type === undefined || schema.type === 'string'
}

export default stringInput
