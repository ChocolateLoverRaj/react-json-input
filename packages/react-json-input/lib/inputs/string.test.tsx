import stringInput from './string'
import { render } from '@testing-library/react'

test('Component', () => {
  expect(render(
    <stringInput.Component
      value='hi'
      {...{} as any}
    />
  ).baseElement).toMatchSnapshot()
})
