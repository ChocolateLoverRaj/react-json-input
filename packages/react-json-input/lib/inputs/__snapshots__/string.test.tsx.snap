// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component 1`] = `
<body>
  <div>
    <tr>
      <td>
        <span>
          âœ“
        </span>
      </td>
      <th
        class="InputName-module__inputName___31Pg-"
      />
      <td>
        <input
          value="hi"
        />
      </td>
      <td />
      <td />
    </tr>
  </div>
</body>
`;
